src/style.css

```diff
@@ -0,0 +1,60 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: inline-block;
+  margin: 0;
+  padding: 0;
+  margin-left: -2px;
+}
+
+.stars__row {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 20px;
+}
+
+.stars--5 .stars__star {
+  background: url(../src/images/star-active.svg) center no-repeat;
+}
+
+.stars--0 .stars__star:nth-child(n+1) {
+  background: url(../src/images/star.svg) center no-repeat;
+}
+
+.stars--1 .stars__star:nth-child(n+2) {
+  background: url(../src/images/star.svg) center no-repeat;
+}
+
+.stars--2 .stars__star:nth-child(n+3) {
+  background: url(../src/images/star.svg) center no-repeat;
+}
+
+.stars--3 .stars__star:nth-child(n+4) {
+  background: url(../src/images/star.svg) center no-repeat;
+}
+
+.stars--4 .stars__star:nth-child(n+5) {
+  background: url(../src/images/star.svg) center no-repeat;
+}
+
+.stars--1 .stars__star:nth-child(1) {
+  background: url(../src/images/star-active.svg) center no-repeat;
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background: url(../src/images/star-active.svg) center no-repeat;
+}
```

Comment:
Combine them with commas, and only one time set` background: url(../src/images/star-active.svg) center no-repeat;`